primitives/staking/src/lib.rs

```diff
@@ -138,7 +138,7 @@ pub trait StakingInterface {
 	/// Get `StakingLedger` by controller account.
 	fn get_ledger(ctrl: &Self::AccountId) -> StakingLedger<Self>;
 
-	/// The ideal number of staking participants.
+	/// The ideal number of active validators.
```

Comment:
@kianenigma I think this reflects the situation better than before, judging by usages of this storage value.